ouroboros-consensus/test-storage/Test/Ouroboros/Storage/ChainDB/Iterator.hs

```diff
@@ -298,8 +300,10 @@ initIteratorEnv TestSetup { immutable, volatile } tracer = do
       where
         epochInfo = fixedSizeEpochInfo epochSize
         isEBB     = const Nothing
-        addDummyBinaryInfo blob = ImmDB.BinaryInfo
-          { ImmDB.binaryBlob   = blob
-          , ImmDB.headerOffset = 0
-          , ImmDB.headerSize   = 0
-          }
+
+addDummyBinaryInfo :: blk -> ImmDB.BinaryInfo blk
```

Comment:
As it is no longer `ImmDB`-specific, can you import `BinaryInfo` from `Ouroboros.Storage.Common` (just `BinaryInfo (..)`) instead and remove the `ImmDB.` qualifier here?